airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""
+Module exposing the tests command to test airbyte-ci projects.
+"""
+
+import asyncclick as click
+from pipelines.cli.lazy_group import LazyGroup
+
+
+@click.group(
+    cls=LazyGroup,
+    help="Commands related to formatting.",
+    lazy_subcommands={
+        "java": "pipelines.airbyte_ci.format.java.commands.java",
+        "js": "pipelines.airbyte_ci.format.js.commands.js",
+        "license": "pipelines.airbyte_ci.format.license.commands.license",
+        "python": "pipelines.airbyte_ci.format.python.commands.python",
+    },
+    invoke_without_command=True,
+)
+@click.option("--fix/--check", type=bool, default=None, help="Whether to automatically fix any formatting issues detected.  [required]")
+@click.pass_context
+async def format(ctx: click.Context, fix: bool):
+    from pipelines.airbyte_ci.format.java.commands import java
+    from pipelines.airbyte_ci.format.js.commands import js
+    from pipelines.airbyte_ci.format.license.commands import license
+    from pipelines.airbyte_ci.format.python.commands import python
```

Comment:
`isort` usually yells at me when I perform inside imports. Can we declare them at the top of the module?

Reply:
It would negate the benefits of lazy importing for this command, but i agree it looks hacky. FWIW, isort doesn't yell about this. I personally don't have a strong opinion either way!